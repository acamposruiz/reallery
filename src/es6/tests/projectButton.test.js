import React from 'react';
import ProjectButton from '../../es6/projectButton.es6';
import renderer from 'react-test-renderer';

test('projectButton', () => {
    const stringParam = {"name": "Ducks", "type": "sculpture"};
    const component = renderer.create(
        <ProjectButton
            view="home"
            lng="en"
            key="0"
            key2="001"
            name="Project name"
            strings={stringParam}
            color="red"
            type="sculpture"
            publish="1"
        />
    );
    let tree = component.toJSON();

    expect(tree).toMatchSnapshot();
});