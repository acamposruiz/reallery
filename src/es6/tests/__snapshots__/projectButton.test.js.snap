// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProjectButton 1`] = `
<div
  className="project-button-container  no-active home-project-button-undefined"
>
  <a
    href="/#/en/project/view/undefined"
  >
    <div
      className="project-icon color-red  icon-ico_undefined"
    >
      <ContentIcons
        dataicon={
          Object {
            "family": "FontAwesome",
            "icon": "FaAsterisk",
          }
        }
      />
    </div>
  </a>
  <div
    className="text title"
  >
    Project name
  </div>
</div>
`;
